### ISSUE TITLE
Error after updating :(  Tried everything, clip is failing.

### ISSUE THREAD

**Issue by MGParisi** [CosmicLaca/ComfyUI_Primere_Nodes#43]
```
E:\ComfyUI>pause
Press any key to continue . . .  Collecting clip
   Using cached clip-0.2.0.tar.gz (5.5 kB)
   Preparing metadata (setup.py): started
   Preparing metadata (setup.py): finished with status 'error'
[!]   error: subprocess-exited-with-error
[!]
[!]   python setup.py egg_info did not run successfully.
[!]   exit code: 1
[!]
[!]   [7 lines of output]
[!]   Traceback (most recent call last):
[!]     File "<string>", line 2, in <module>
[!]     File "<pip-setuptools-caller>", line 34, in <module>
[!]     File "C:\Users\mgpar\AppData\Local\Temp\pip-install-rogd_kli\clip_{some random string}\setup.py", line 24, in <module>
[!]       version=clip.__version__,
[!]               ^^^^^^^^^^^^^^^^
[!]   AttributeError: module 'clip' has no attribute '__version__'
[!]   [end of output]
[!]
[!]   note: This error originates from a subprocess, and is likely not a problem with pip.
[!] error: metadata-generation-failed
[!]
[!] Encountered error while generating package metadata.
[!]
[!] See above for output.
[!]
[!] note: This is an issue with the package mentioned above, not pip.
[!] hint: See above for details.
install/(de)activation script failed: E:\ComfyUI\ComfyUI\custom_nodes\ComfyUI_Primere_Nodes

## ComfyUI-Manager: EXECUTE => ['E:\\ComfyUI\\python_embeded\\python.exe', '-m', 'pip', 'install', 'opencv_contrib_python']

## Execute install/(de)activation script for 'E:\ComfyUI\ComfyUI\custom_nodes\ComfyUI_Primere_Nodes'

## ComfyUI-Manager: EXECUTE => ['E:\\ComfyUI\\python_embeded\\python.exe', '-m', 'pip', 'install', 'opencv_python']

## Execute install/(de)activation script for 'E:\ComfyUI\ComfyUI\custom_nodes\ComfyUI_Primere_Nodes'

## ComfyUI-Manager: EXECUTE => ['E:\\ComfyUI\\python_embeded\\python.exe', '-m', 'pip', 'install', 'opencv_python_headless']

## Execute install/(de)activation script for 'E:\ComfyUI\ComfyUI\custom_nodes\ComfyUI_Primere_Nodes'
```

**Comment by MGParisi** [comment]
I've tried everything to solve this.  I suspect it is a dependency issue, and that as soon as you upgrade your comfy_ui you will encounter this same mess.  I tried everything, but irregardless of what I do ComfyUI has a dependency issue.  And the issues are unresolveable in stand alone version.  I've gotten past the clip issue, but then I get issues with other modules like manager and others.

```
------------------------------------------
Adding E:\ComfyUI_windows_portable\ComfyUI\custom_nodes to sys.path
Could not find efficiency nodes
Loaded ControlNetPreprocessors nodes from E:\ComfyUI_windows_portable\ComfyUI\custom_nodes\comfyui_controlnet_aux
Could not find AdvancedControlNet nodes
Could not find AnimateDiff nodes
Could not find IPAdapter nodes
Could not find VideoHelperSuite nodes
### Loading: ComfyUI-Impact-Pack (V8.0.1)
Loaded ImpactPack nodes from E:\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-Impact-Pack
[Impact Pack] Wildcards loading done.
Traceback (most recent call last):
  File "E:\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI_Primere_Nodes\components\sana\diffusion\model\builder.py", line 26, in <module>
    from mmengine.registry import Registry
ModuleNotFoundError: No module named 'mmengine'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "E:\ComfyUI_windows_portable\ComfyUI\nodes.py", line 2072, in load_custom_node
    module_spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "E:\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI_Primere_Nodes\__init__.py", line 7, in <module>
    from .Nodes import Dashboard
  File "E:\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI_Primere_Nodes\Nodes\Dashboard.py", line 57, in <module>
    from ..components.sana.diffusion.model.builder import build_model
  File "E:\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI_Primere_Nodes\components\sana\diffusion\__init__.py", line 6, in <module>
    from .dpm_solver import DPMS
  File "E:\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI_Primere_Nodes\components\sana\diffusion\dpm_solver.py", line 20, in <module>
    from .model.dpm_solver import DPM_Solver, NoiseScheduleFlow, NoiseScheduleVP, model_wrapper
  File "E:\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI_Primere_Nodes\components\sana\diffusion\model\dpm_solver.py", line 23, in <module>
    from .nets.sana_blocks import (
  File "E:\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI_Primere_Nodes\components\sana\diffusion\model\nets\__init__.py", line 1, in <module>
    from .sana import (
  File "E:\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI_Primere_Nodes\components\sana\diffusion\model\nets\sana.py", line 28, in <module>
    from ..builder import MODELS
  File "E:\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI_Primere_Nodes\components\sana\diffusion\model\builder.py", line 28, in <module>
    from mmcv import Registry
ModuleNotFoundError: No module named 'mmcv'
```

I've tried re-install comfy, but that doesn't work.  That got me here.  I've installed both MMCV and mmengine.  But they are throwing conflicts in the dependancies of torch, pip, and others.

**Comment by MGParisi** [comment]
I'm about to disable Primere as its the only one failing now.  I will have to redo my entire workflow.  This is a devistating bug, and I'm so sorry I upgraded.

**Comment by CosmicLaca** [comment]
@MGParisi Because the new dev support Sana, some undocumented upgrade needed. The nodepack called `Extra Models` required for Sana (and some other extras), and just use Primere's requirements.txt after update (`pip install -r requirements.txt`). For me working on Python 3.10 and 3.12.

![2024-12-18_21-50-43](https://github.com/user-attachments/assets/891edc50-6453-4531-bc99-31109b63fcf0)

**Comment by MGParisi** [comment]
Thank you for your help.

That did something, 
```
ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behaviour is the source of the following dependency conflicts.
clip-by-openai 1.1 requires torch<1.7.2,>=1.7.1, but you have torch 2.5.1 which is incompatible.
clip-by-openai 1.1 requires torchvision==0.8.2, but you have torchvision 0.20.1 which is incompatible.
xformers 0.0.27.post2 requires torch==2.4.0, but you have torch 2.5.1 which is incompatible.
```

**Comment by CosmicLaca** [comment]
@MGParisi This is the issue of incompatible versions of several Python libs. But as I see this is only the version problem of `clip` library. This lib I used only at aesthetic scorer node, what I dont really liked because over overestimates the anime pics. So I changed this to another model, and for new model no need the clip library. Update my nodepack only and try to start comfy, the clip library import already deleted.
